docs/apm/troubleshooting.asciidoc

```diff
@@ -98,11 +98,6 @@ If you feel like you'd be losing valuable information by following this naming c
 You can always add additional metadata to your transactions using {apm-guide-ref-v}/metadata.html#labels-fields[labels] (indexed) or
 {apm-guide-ref-v}/metadata.html#custom-fields[custom context] (non-indexed).
 
-After ensuring you've correctly named your transactions,
-you might still see an error in the APM app related to too many transaction names.
-If this is the case, you can increase the default number of transaction groups displayed in the APM app by configuring
-<<apm-settings-kb,`xpack.apm.ui.transactionGroupBucketSize`>>.
```

Comment:
@achyutjhunjhunwala @yngrdyn Please coordinate on this error message. Users may still run into limitations. They can work around the Kibana limit (10.000) by using the search bar in the UI. 
They can work around the APM Server limit by increasing it in the APM Server config.

I also added some more context here: https://github.com/elastic/kibana/pull/148641